### ISSUE TITLE
Change URL structure to a better readable solution

### ISSUE THREAD

**Issue by sapkra** [storybooks/storybook#3837]
### Issue details

We want to link on our stories in our JIRA tickets for our QA.
It's really hard to create short readable URLs, so my suggestion is to create a new URL structure.

> /?selectedKind=Organisms%2FProductOverview&selectedStory=Filter&someAddonConfig=optional

should be restructured to

> /Organisms/ProductOverview/Filter?someAddonConfig=optional

### Steps to reproduce

You just have to create a story in a hierarchy tree.

### Affected platforms 

- All browsers

### Recreated this ticket because it was automatically closed and it's still relevant.
Old ticket #2466 
It would be great to just reopen the old ticket and close this one.

**Comment by ndelangen** [comment]
On the roadmap

**Comment by sapkra** [comment]
It's still relevant. Is it possible to disable this `inactive` feature somehow?

**Comment by trentrand** [comment]
@ndelangen Are you sure about that? I just checked and didn't see it on the roadmap.

**Comment by ndelangen** [comment]
@trentrand will be part of 4.0.0 via this PR:
https://github.com/storybooks/storybook/pull/4086

**Comment by tmeasday** [comment]
This is done in v5 branch.

**Comment by shilman** [comment]
Released: https://github.com/storybooks/storybook/releases/tag/v5.0.0-alpha.0
